frontend/src/components/world/MazeGameInvite.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ReactNode } from 'react';
+import { useToast, Button, SimpleGrid } from '@chakra-ui/react';
+
+export default function MazeGameInvite(): JSX.Element {
+  const INVITING_PLAYER = 'player1'; // TODO: This is just a placeholder, remove when invite logic gets added
+  const TOAST_ID = 'invite_player';
+  const toast = useToast();
+
+  function acceptRace() {
+    toast.close(TOAST_ID);
+    // TODO: add logic to start game
+  }
+
+  function rejectRace() {
+    toast.close(TOAST_ID);
+    toast({
+      title: "Race rejected",
+      description: "Maybe next time :(",
+      status: "info",
+      duration: 2000,
+      isClosable: true
+    });
+    // TODO: add logic to inform other player that the race has been rejected
+  }
+
+  function makeButtons(): ReactNode {
+    return (<>
+      <SimpleGrid columns={2} spacing={10}>
+        <Button colorScheme="blackAlpha" onClick={acceptRace}>accept</Button>
+        <Button colorScheme="whiteAlpha" onClick={rejectRace}>reject</Button>
+      </SimpleGrid>
+    </>)
+  }
+
+  // TODO: currently using a button to show the toast message, remove when invite logic gets added
+  return (
+    <Button
+      onClick={() =>
+        toast({
+          id: TOAST_ID,
+          title: `${INVITING_PLAYER} has invited you to race, accept?`,
+          description: makeButtons(),
+          duration: null,
```

Comment:
Personally I would set a long-ish timeout duration (maybe 10s?) on the race invite so that the other guy doesn't wait around forever for a response. Doesn't have to be done in this ticket, but something to think about!

Reply:
Good idea.  Just to keep this sprint from being delayed any longer I will merge for now, and it can be implemented with the invite logic.